frontend/src/store/slices/accessRequests/accessRequestsActionCreator.spec.tsx

```diff
@@ -26,78 +27,57 @@ const getWrapper = (store: any) => ({ children }: any) => (
+      fetchCurrentAccessRequest();
+      await waitFor(async () => expect(mockAxios.history.get[0]).toMatchObject({ url: url }));
     });
-    it('Request successful, dispatches success with correct response', () => {
+    it('Request successful, dispatches success with correct response', async () => {
       mockAxios.onGet(url).reply(200, mockResponse);
-      renderHook(
-        () =>
-          useAccessRequests()
-            .fetchCurrentAccessRequest()
-            .then(() => {
-              expect(
-                find(currentStore.getActions(), { type: 'network/logRequest' }),
-              ).not.toBeNull();
-              expect(
-                find(currentStore.getActions(), { type: 'network/logSuccess' }),
-              ).not.toBeNull();
-              expect(currentStore.getActions()).toContainEqual({
-                payload: mockResponse,
-                type: 'accessRequests/storeAccessRequest',
-              });
-            }),
-        {
-          wrapper: getWrapper(getStore()),
-        },
-      );
+      const { fetchCurrentAccessRequest } = setup();
+      fetchCurrentAccessRequest();
+      await waitFor(async () => {
+        expect(find(currentStore.getActions(), { type: 'network/logRequest' })).not.toBeNull();
+        expect(find(currentStore.getActions(), { type: 'network/logSuccess' })).not.toBeNull();
+        expect(currentStore.getActions()).toContainEqual({
+          payload: mockResponse,
+          type: 'accessRequests/storeAccessRequest',
+        });
+      });
     });
 
-    it('Request failure, dispatches error with correct response', () => {
+    it('Request failure, dispatches error with correct response', async () => {
       mockAxios.onGet(url).reply(400, MOCK.ERROR);
-      renderHook(
-        () =>
-          useAccessRequests()
-            .fetchCurrentAccessRequest()
-            .then(() => {
-              expect(
-                find(currentStore.getActions(), { type: 'network/logRequest' }),
-              ).not.toBeNull();
-              expect(find(currentStore.getActions(), { type: 'network/logError' })).not.toBeNull();
-              expect(currentStore.getActions()).not.toContainEqual({
-                payload: {
-                  data: {
-                    items: [mockAccessRequest],
-                    page: 1,
-                    pageIndex: 0,
-                    quantity: 0,
-                    total: 0,
-                  },
-                },
-                type: 'accessRequests/storeAccessRequest',
-              });
-            }),
-        {
-          wrapper: getWrapper(getStore()),
-        },
-      );
+      const { fetchCurrentAccessRequest } = setup();
+      fetchCurrentAccessRequest();
+      await waitFor(async () => {
+        expect(find(currentStore.getActions(), { type: 'network/logRequest' })).not.toBeNull();
+        expect(find(currentStore.getActions(), { type: 'network/logError' })).not.toBeNull();
```

Comment:
This assertion can lead to false-positives because:

1. lodash [find](https://lodash.com/docs/4.17.15#find) returns **undefined** if it cannot find the element
2. **undefined** is not null so it will pass

I changed these types of assertions to **not.toBeUndefined()** or **toBeDefined()** in my previous PR to solve that